Updates for SDK X 10.3.0 (#107)
helpshift/helpshift-docs: docs/sdkx_android/support-tools.mdx

For more info about this API and languages you can refer [here](/sdkx_android/i18n/).

## Getting SDK Version {#get-sdk-version}

> You can use the API `getSDKVersion` to get the current version of Helpshift SDK.

Reviewer: ```suggestion
You can use `getSDKVersion` API to get the current version of Helpshift SDK X.
```

<Admonition type="info" title="Note">

> This API is available only from 10.3.0 versions

Reviewer: ```suggestion
This API is available from 10.3.0+ version
```

</Admonition>

```java
  // Returns SDK version as a string eg. "10.3.0"
  Helpshift.getSDKVersion();
```

## Clossing a Helpshift Session {#close-session}

> You can use the API `Helpshift.closeSession` to close an active SDK session. If no SDK session is active, calling this API will have no effect.

Reviewer: ```suggestion
You can use `Helpshift.closeSession` API to close an active SDK session. If no SDK session is active, calling this API will have no effect.
```

<Admonition type="info" title="Note">

> This API is available from 10.3.0 version

Reviewer 2: ```suggestion
This API is available from 10.3.0 version.
```

> This API is available only from 10.3.0 versions

Reviewer: ```suggestion
This API is available from 10.3.0 version
```
